app/src/db/migrations/20240203003345_last-modified-last-synced.js

```diff
@@ -0,0 +1,32 @@
+exports.up = function (knex) {
+  return Promise.resolve()
+    // Add lastModifiedDate column to version table
+    .then(() => knex.schema.alterTable('version', table => {
+      table.timestamp('lastModifiedDate', { useTz: true });
+    }))
+    // Add lastSyncedDate column to object table
+    .then(() => knex.schema.alterTable('object', table => {
+      table.timestamp('lastSyncedDate', { useTz: true });
+    }))
+    // Add lastSyncRequesteDate to bucket table
+    .then(() => knex.schema.alterTable('bucket', table => {
+      table.timestamp('lastSyncRequestedDate', { useTz: true });
+    }));
+
+};
+
+exports.down = function (knex) {
+  return Promise.resolve()
+    // Drop lastModifiedDate column from version table
+    .then(() => knex.schema.alterTable('version', table => {
+      table.dropColumn('lastModifiedDate');
+    }))
+    // DroplastSyncedDate column from object table
+    .then(() => knex.schema.alterTable('object', table => {
+      table.dropColumn('lastSyncedDate');
+    }))
+    // Drop lastSyncRequesteDate from bucket table
+    .then(() => knex.schema.alterTable('bucket', table => {
+      table.dropColumn('lastSyncRequestedDate');
+    }));
+};
```

Comment:
its good practice to list these changes in the reverse of their order in the up process.
here it doesnt matter much.. but if you have foreign keys or dependencies, they may need to be removed in the a certain order.